napari/_app_model/context/_context.py

```diff
@@ -16,6 +17,32 @@
 __all__ = ['create_context', 'get_context', 'Context', 'SettingsAwareContext']
 
 
+class ContextMapping(collections.abc.Mapping):
+    def __init__(self, initial_values: collections.abc.Mapping):
+        self._base_store = initial_values
```

Comment:
I think it's odd for the `_base_store` to be assigned `initial_values`, because it makes it seem like not a mapping. Can we call it `initial_context_mapping` or something like that?